cartoframes/io/managers/context_manager.py

```diff
@@ -43,6 +43,21 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def not_found(func):
+    def decorator_func(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+
+        except CartoException as e:
```

Comment:
I would simplify this to

```py
except NotFoundException:
    raise Exception('Resource not found')
```

Reply:
That doesn't work :(

The exception is a `CartoException`, created with a `NotFoundException` as the only parameter, so we need to inspect the `args`.